README.md

```diff
@@ -89,6 +97,25 @@ return assert.isRejected(promise, Error, "optional message");
 return assert.isRejected(promise, /error message matcher/, "optional message");
 ```
 
+### Multiple Promise assertions
```

Comment:
This section already exists: "Another example of where this can be useful is when performing assertions on multiple promises..."